classes/student.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+
+# Class for students, inherits from person
+class Student < Person
+  def initialize(name, age, parent_permission, classroom)
```

Comment:
- [ ] Please make sure that the `name` and `parent_permission` are optional and have default values of "Unknown" and `true` respectively. :pray: